atomicapp/cli/main.py

```diff
@@ -483,7 +483,7 @@ def run(self):
         cmdline.insert(0, args.action)  # Place 'action' at front
 
         # Finally, parse args and give error if necessary
-        args = self.parser.parse_args(cmdline)
+        args, unknown = self.parser.parse_known_args(cmdline)
```

Comment:
instead of doing this why don't we just delete the `--mode` arg from args on line 482